pkg/azurelustreplugin/entrypoint.sh

```diff
@@ -80,63 +118,42 @@ if [[ "${installClientPackages}" == "yes" ]]; then
 
   echo "$(date -u) Installed Lustre client packages."
 
-  # Issue #115 Remove workaround for LNET fix
-  # Revert below LNET fix, please don't remove the lines to cleanup rule files
-
   init_lnet="true"
-  
+
   if lsmod | grep "^lnet"; then
     if lnetctl net show --net tcp | grep interfaces; then
-      echo "$(date -u) LNet is loaded skip the load."
+      echo "$(date -u) LNet is loaded skip the load"
+      echo "$(date -u) Adding missing interfaces"
+      add_net_interfaces
```

Comment:
LNet module is loaded in L138. So, the `lnetctl` command in `add_net_interfaces` will be executed before LNet is loaded. And will encounter some error I think.

Reply:
There are two logical flows here, one if the pod is being run on a node with lnet/lnetctl already loaded, or not.

The if statement here means this will run if lnet/lnetctl is already loaded (L123-124). This just makes sure that all available ethernet nics are added.

Line 138, which loads lnet, is inside a conditional to only run if lnet hasn't already been loaded, specifically when init_lnet is true (this block makes sure that's skipped on L128).